src/Commands/BuildToolsCommand.php

```diff
@@ -658,9 +659,13 @@ protected function expandSourceAliases($source)
      */
     protected function autodetectUpstream($siteDir)
     {
-        return 'Empty Upstream';
-        // or 'Drupal 7' or 'WordPress'
-        // return 'Drupal 8';
+        if (file_exists($siteDir . '/web/wp-config.php')) {
```

Comment:
I think we should use something like `find -maxdepth 2 -type f -name 'wp-config.php` rather than hard-coding directories or guessing what subdirectory will have WordPress core.